ini_multiline_value_fix_test.go

```diff
@@ -0,0 +1,60 @@
+package ini_test
+
+import (
+	"gopkg.in/ini.v1"
+	. "github.com/smartystreets/goconvey/convey"
+	"path/filepath"
+	"testing"
+)
+
+type testData struct {
+	Value1 string `ini:"value1"`
+	Value2 string `ini:"value2"`
+}
+
+func TestMultiline(t *testing.T) {
+	Value1 := "some text here\n"+
+		"some more text here\n"+
+		"\n"+
+		"there is an empty line above and below\n"
+
+	Value2 := "there is an empty line above\n"+
+		"that is not indented so it should not be part\n"+
+		"of the value"
+
+	Convey("Parse Python-style multiline values", t, func() {
```

Comment:
Comments inside this func feel too verbose, please refer to tests in other source files to clean up a bit.

Reply:
Which comments?

Reply:
Hmm, I would say all the comments in the test convey.